.github/workflows/update_disabled_tests.yml

```diff
@@ -20,6 +20,19 @@ jobs:
         run: |
           cat disabled-tests.json
       - name: Push file to test-infra repository
+        uses: dmnemec/copy_file_to_another_repo_action@eebb594efdf52bc12e1b461988d7254322dac131
+        env:
+           API_TOKEN_GITHUB: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          source_file: 'disabled-tests.json'
+          destination_repo: 'pytorch/test-infra'
+          destination_folder: 'stats'
+          destination_branch: slow-disabled-tests
```

Comment:
Could we have a more general branch name for prosperity?

Reply:
From this list which sounds better?
1. stats-update
2. generated-stats 
3. generated-files

Reply:
I like the second or first